docs/en/guides/pipeline-integrations/administration/advanced-toolkit-administration.md

```diff
@@ -0,0 +1,1526 @@
+ - /mnt/projects/big_buck_bunny/assets/Character/Squirrel/Surface/reference/footage
+ - /mnt/projects/big_buck_bunny/assets/Character/Squirrel/Surface/review
+ - /mnt/projects/big_buck_bunny/assets/Character/Squirrel/Surface/work
+ - /mnt/projects/big_buck_bunny/assets/Character/Squirrel/Surface/work/images
+ - /mnt/projects/big_buck_bunny/assets/Character/Squirrel/Surface/work/maya
+ - /mnt/projects/big_buck_bunny/assets/Character/Squirrel/Surface/work/maya/workspace.mel
+ - /mnt/projects/big_buck_bunny/assets/Character/Squirrel/Surface/work/maya/snapshots
+ - /mnt/projects/big_buck_bunny/assets/Character/Squirrel/Surface/work/nuke
+ - /mnt/projects/big_buck_bunny/assets/Character/Squirrel/Surface/work/nuke/snapshots
+ - /mnt/projects/big_buck_bunny/assets/Character/Squirrel/Surface/work/photoshop
+ - /mnt/projects/big_buck_bunny/assets/Character/Squirrel/Surface/work/photoshop/snapshots
+
+In total, 93 folders were processed.
+```
+### shell
+
+Starts an interactive Python shell in your currently selected context. Sets up handy references to pre-initialized APIs, context, and engine objects. You can use `./tank shell` to try out Toolkit and {% include product %} API code examples. The tank executable is located in the **Pipeline Configuration root** on your local machine, and looks something like:
+
+```
+> cd /my_tank_configs/project_foobar
+```
+
+Example output from this command:
+
+```python
+Welcome to the {% include product %} pipeline toolkit!
+For documentation, see https://support.{% include product %}software.com
+You are running a tank command associated with {% include product %} Project 'Chasing the
+Light'. Only items associated with this project will be considered.
+- Found Shot moo87 (Project 'Chasing the Light')
+- Starting the {% include product %} pipeline toolkit v0.15.14.
+- Setting the Context to Shot moo87.
+- Started Shell Engine version v0.3.3
+- Environment: /mnt/software/tank/chasing_the_light/config/env/shot.yml.
+- Running shell...
+
+Welcome to {% include product %} Pipeline Toolkit Python!
+2.7.1 (r271:86832, Jul 31 2011, 19:30:53)
+[GCC 4.2.1 (Based on Apple Inc. build 5658) (LLVM build 2335.15.00)]
+Running on darwin
+
+- A tk API handle is available via the tk variable
+- A {% include product %} API handle is available via the {% include product %} variable
+- Your current context is stored in the context variable
+- The shell engine can be accessed via the engine variable
+>>>
+>>> tk
+<Sgtk Core vHEAD@0x105f66990 Config /mnt/software/tank/chasing_the_light>
+>>>
+>>> context
+<Sgtk Context:   Project: {'type': 'Project', 'id': 88, 'name': 'Chasing the Light'}
+  Entity: {'type': 'Shot', 'id': 1184, 'name': 'moo87'}
+  Step: None
+  Task: None
+  User: {'type': 'HumanUser', 'id': 42, 'name': 'Manne \xc3\x96hrstr\xc3\xb6m'}
+  Additional Entities: []>
+>>>
+>>> {% include product %}
+<tank_vendor.{% include product %}_api3.{% include product %}.{% include product %} object at 0x105fb2990>
```

Comment:
These names have not changed in the API. When in code, everything is still called `Shotgun`